test/spec/services/angular-apimock.js

```diff
@@ -92,9 +92,10 @@ describe('Service: apiMock', function () {
 			$httpBackend.expect(defaultExpectMethod, defaultExpectPath).respond(404);
 
 			$http(defaultRequest)
-				.success(function () {
-					fail(); // Todo: How to fail the test if this happens?
-					failCb && failCb();
+				.success(/* istanbul ignore next - it is expecting a failure, so should fail tests if happen */
```

Comment:
It is just an Istambul directive to ignore the whole success function that should never be invoked, it is because if it is invoked it is not expected to happen and test should fail.
Directly related to including tests in coverage.